### ISSUE TITLE
images are flickering

### ISSUE THREAD

**Issue by nsklaus** [acg/w3m#1]
i tried a few terminals that are images capables, (st, kitty, xterm, uxterm..)
and i see the same problem with them all when using w3m.
the images are flickering when i move the cursor.
from what i've read it may be a probem caused by font antialiasing causing refreshes.

since antialiasing is everywhere nowadays maybe w3m could develop a workaround about this problem ?

**Comment by naowalrahman** [comment]
Yeah, same thing happens to me. When I open up the browser, images aren't shown, but when i try scrolling down using the cursor, the image constantly flickers but just stops showing when I stop moving the cursor. I'm using the kitty terminal.

**Comment by newptcai** [comment]
Same thing on Ubuntu 24.04. Has anyone found a way to make it work?

**Comment by officialBeebe** [comment]
I'm having the same issue on arch btw using st. It will work with xterm.

**Comment by naowalrahman** [comment]
> I'm having the same issue on arch btw using st. It will work with xterm.

SIXEL is the standard nowadays for image previewing in terminal emulators, consider switching to a terminal that supports it (kitty, foot, wezterm, etc.)

**Comment by naowalrahman** [comment]
> I'm having the same issue on arch btw using st. It will work with xterm.

SIXEL is the standard nowadays for image previewing in terminal emulators, consider switching to a terminal that supports it (kitty, foot, wezterm, etc.)

Also this post gives me nostalgia, I was so new to Linux 4 years ago and found it all so cool :-)

**Comment by officialBeebe** [comment]
> SIXEL is the standard nowadays for image previewing in terminal emulators

I'm using i3 and just dumped kitty (I wanted it for the name 😿) and settled with xterm AND JUST got it configured! I'll check that out though. Thanks for the tip!

> I was so new to Linux 4 years ago and found it all so cool :-)

That is about where I am right now. I've had a lot of fun with linux for development. I thought after a year now it was time to really dive in and really *master* Linux - thus began the Arch journey. Discovering how it all pieces together is so fascinating.
